rplugin/python/ensime.py

```diff
@@ -172,13 +171,9 @@ def au_vim_leave(self, *args, **kwargs):
     def au_buf_leave(self, *args, **kwargs):
         super(NeovimEnsime, self).au_buf_leave(*args, **kwargs)
 
-    @neovim.autocmd('CursorHold', **autocmd_params)
-    def au_cursor_hold(self, *args, **kwargs):
-        super(NeovimEnsime, self).au_cursor_hold(*args, **kwargs)
-
-    @neovim.autocmd('CursorMoved', **autocmd_params)
-    def au_cursor_moved(self, *args, **kwargs):
-        super(NeovimEnsime, self).au_cursor_moved(*args, **kwargs)
+    @neovim.function('EnRefreshMessages', sync=True)
```

Comment:
This is only for Neovime, you should also define this function for Vim in `plugin/ensime.vim`